theseus_gui/src-tauri/src/api/settings.rs

```diff
@@ -45,6 +45,14 @@ pub async fn settings_get() -> Result<FrontendSettings> {
     Ok(frontend_settings)
 }
 
+// Gets just the theme
+// invoke('theme_get')
+#[tauri::command]
+pub async fn theme_get() -> Result<Theme> {
+    let backend_settings = settings::get().await?;
+    Ok(backend_settings.theme)
+}
+
```

Comment:
I had mentioned having a call to just get the theme on App.vue since the settings_get is currently an expensive read. I didn't want to have the launcher need to take 5-10 seconds to load on Win10.